packages/nimbuild-corejs/lib/index.js

```diff
@@ -30,7 +34,10 @@ async function getPolyfillString({
     uaString
 }) {
     // Resolve base feature set based on include / exclude
-    const features = getBaseFeatureModules({include, exclude});
+    const features = getBaseFeatureModules({
```

Comment:
Are these formatting changes driven by prettier?  When I edit this file, formatting snaps back to single line.